Contribute/how-to-write-use-markdown.md

```diff
@@ -290,6 +316,36 @@ You can choose from four types of note blocks to draw attention to specific cont
 
 In general, note blocks should be used sparingly because they can be disruptive. Although they also support code blocks, images, lists, and links, try to keep your note blocks simple and straightforward.
 
+Examples:
```

Comment:
This addition introduces duplication of the following section:
https://docs.microsoft.com/en-us/contribute/markdown-reference#alerts-note-tip-important-caution-warning